dts/arm/microchip/mec1727nsz.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+
+#include <dt-bindings/i2c/i2c.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/clock/mchp_xec_pcr.h>
+#include <dt-bindings/interrupt-controller/mchp-xec-ecia.h>
+
+#include "mec172xnsz.dtsi"
+#include "mec172x/mec172x-vw-routing.dtsi"
+#include "mec172x/mec172xnsz-pinctrl.dtsi"
+
+/ {
+	flash1: flash@60000000 {
+		reg = <0x60000000 0x80000>;
+	};
+};
+
+&spi0 {
+	status = "okay";
+	clock-frequency = <12000000>;
+	lines = <2>;
+	port-sel = <2>;
+	chip-select = <0>;
+	pinctrl-0 = < &gpspi_cs_n_gpio116
+		      &gpspi_clk_gpio117
+		      &gpspi_io0_gpio074
+		      &gpspi_io1_gpio075
+		      &gpspi_wp_n_gpio076 >;
+	pinctrl-names = "default";
+
+	int_flash: sst25pf040@0 {
+		compatible ="jedec,spi-nor";
+		/*  4194304 bits = 512K Bytes */
+		size = <0x400000>;
```

Comment:
how about using DT_SIZE_M(4)? (and updating the comment specifying that this is 4Mbit

Reply:
I tried but I get `parse error: malformed value`
do you have an example of the usage?

Reply:
fixed